### ISSUE TITLE
Mobile: Long title fails "mobile-friendly test" | Truncate title on Channel Page

### ISSUE THREAD

**Issue by infinite-persistence** [lbryio/lbry-desktop#6635]
It actually works fine on Firefox Android, but we shouldn't display super-long titles in the first place.  Truncate with ellipsis?

![image](https://user-images.githubusercontent.com/64950861/126584515-3f73ddb2-a745-40e7-a3ff-2bb0e58d833d.png)

<img src="https://user-images.githubusercontent.com/64950861/126584682-20e32479-6ad5-46a2-9f05-13cdbf9bcfeb.png" width="400">

**Comment by infinite-persistence** [comment]
Fixed by #6470.

![image](https://user-images.githubusercontent.com/64950861/127617754-1af5c287-c1dd-4a30-a1b4-7a3b960fb5b7.png)
